.github/workflows/android.yml

```diff
@@ -0,0 +1,32 @@
+name: Android
+
+on: [push, pull_request]
+
+jobs:
+  linux:
+    strategy:
+      fail-fast: false
+      matrix:
+        image:
+          - r22beta1
+        build_type: [Debug, Release]
+        android_abi:
+          - armeabi-v7a
+          - arm64-v8a
+        android_platform:
+          - android-26
+
+    runs-on: ubuntu-latest
+
+    container:
```

Comment:
@d-frey Where can I find the documentation on using the docker on GitHub-Actions ? ... I didn't find one :-( I need to set some "parameters" on docker "run"...

Reply:
I'm not experienced with any of that, but maybe this might help: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer

Reply:
Also, check out our linux.yml, it is using containers from Conan where we also need to set some options.

Reply:
> but maybe this might help: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer

Yep, this was the right hint :-) many thanks. 

> Also, check out our linux.yml, it is using containers from Conan where we also need to set some options.

Yep I know. A copy from the linux.yml and past it in android.yml :-)